_data/pratapnea/pratapnea.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Pratap Neaupane # No longer than 18 characters
+institution: Nepathya College 🚩 # no longer than 58 characters
+profile_pic: pratapnea.jpg # Name and extension of your profile picture(ex. mona.png)
+quote: "Everyone has thier own role in someone's life and some people are really great to meet. Thankyou everyone!" # no longer than 100 characters
```

Comment:
Must **not** be longer than 100 characters. Yours is **108** characters.

Typos present:
* thier → their
* Thankyou → Thank you